File: 1610-xor-operation-in-an-array/xor-operation-in-an-array.js
var xorOperation = function(n, start) {
    let nums = [];
    for(let i = 0; i < n; i++){
        nums.push(start + 2 * i);
    }
    output = nums[0];
    for (let i = 1; i < nums.length; i ++){
        output ^= nums[i];
    }
    return output;    
};

File: 1610-xor-operation-in-an-array/xor-operation-in-an-array.py
class Solution:
    def xorOperation(self, n: int, start: int) -> int:
        nums = []
        for i in range(0, n):
            nums.append(start + 2 * i)
        output = nums[0]
        for i in range(1,len(nums)):
            output = output ^ nums[i]
        return output
